airflow/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -162,7 +162,7 @@ def _refresh_credentials(self) -> Dict[str, Any]:
             raise RuntimeError(f'sts_response_http_status={sts_response_http_status}')
         credentials = sts_response['Credentials']
         expiry_time = credentials.get('Expiration').isoformat()
-        self.log.info(f'New credentials expiry_time:{expiry_time}')
+        self.log.debug(f'New credentials expiry_time:{expiry_time}')
```

Comment:
Unrelated but should this be 
```suggestion
        self.log.debug('New credentials expiry_time: %s', expiry_time)
```

Reply:
Looks good to me, I've updated the PR with your suggested change :+1: